approach_mission.md

```diff
@@ -0,0 +1,164 @@
+# Approaching a Task - determining a website's technical structure. 
+
+When you need to deal with new task, sometimes things seem confusing and even a simple task may be overwhelming because you don’t know from where to start. Well, no need to reinvent the wheel, once you adopt a clear approach to a problem, with defined rules, ever the most complex task becomes approachable and less threatening.
+
+In this post we present a method that includes a few steps of logical thinking to help you find the starting point from which to approach a task. A task being the determination of a website's technical structure.
+
+Task composed of many small parts that if we will map them correctly, then their implementation will be much more accurate and simple. So the big trick is to be able to recognize the entities and the relationships between them. It is ultimately a world of circles and arrows.
+
+Take a look at the following simple story: 
+
+
+> An author writes articles.
+
+
+Let's try to identify the entities and relationships in this story. The best way to do it - is to draw it! Use circles for entities and lines for relationships.
+
+![](111.jpg)
```

Comment:
can you change the file names to make more sense (`111`, and `888`). You can also give them names if you prefer